package.json

```diff
@@ -2,36 +2,33 @@
   "name": "@github/textarea-autosize",
   "description": "Autosizes textarea to size of it's contents.",
   "version": "0.1.3",
-  "main": "dist/textarea-autosize.umd.js",
-  "module": "dist/textarea-autosize.esm.js",
-  "types": "index.d.ts",
+  "main": "dist/textarea-autosize.js",
+  "module": "dist/textarea-autosize.js",
+  "types": "dist/textarea-autosize.d.ts",
   "license": "MIT",
   "repository": "github/textarea-autosize",
   "files": [
-    "dist",
-    "index.d.ts"
```

Comment:
This types file is generated by `tsc` because we enable the `"declaration": true` compiler option. We should delete our index.d.ts file now that the compiler emits one for us.